src/Validation/formValidationSchema.ts

```diff
@@ -73,18 +69,15 @@ const formValidationSchema = [
       .min(3, "Must contain at least 3 or more characters")
       .required("Required"),
     foundingYear: Yup.string()
-      .matches(onlyDigitRegex, "Must contain digits only")
+      .matches(yearRegex, "Please provide a year between 1900-2022")
```

Comment:
Are you going to update that in couple months to 1900-2023?
yup.number().integer().min(1900).min(getCurrentYear())